pages/resources/collectibles.mdx

```diff
@@ -0,0 +1,802 @@
+| ----- | ------- | --------------------------------------- |
+| x     | integer | The x-coordinate of the animation frame |
+| y     | integer | The y-coordinate of the animation frame |
+
+###### Profile Effect Source Structure
+
+| Field | Type   | Description                                     |
+| ----- | ------ | ----------------------------------------------- |
+| src   | string | The URL of the animation image (in APNG format) |
+
+###### Profile Effect Animation Type
+
+| Value | Name         | Description                        |
+| ----- | ------------ | ---------------------------------- |
+| 0     | UNSPECIFIED  | The animation type is unspecified  |
+| 1     | PERSISTENT   | The animation type is persistent   |
+| 2     | INTERMITTENT | The animation type is intermittent |
+
+###### Nameplate Collectible Item Structure
+
+| Field   | Type                                         | Description                                                  |
+| ------- | -------------------------------------------- | ------------------------------------------------------------ |
+| type    | integer                                      | The [collectible product type](#collectible-product-type)    |
+| id      | snowflake                                    | The ID of the nameplate                                      |
+| sku_id  | snowflake                                    | The SKU ID of the nameplate                                  |
+| palette | string                                       | The nameplate's [color palette](#nameplate-color-palette)    |
+| asset   | string                                       | The [nameplate asset path](/reference#cdn-formatting)        |
+| assets  | [item assets](#item-assets-structure) object | The URLs for the static and animated images of the nameplate |
+| label   | string                                       | The nameplate accessibility description                      |
+
+###### Nameplate Color Palette
+
+| Value      | Name      |
+| ---------- | --------- |
+| none       | None      |
+| crimson    | Crimson   |
+| berry      | Berry     |
+| sky        | Sky       |
+| teal       | Teal      |
+| forest     | Forest    |
+| bubble_gum | BubbleGum |
+| violet     | Violet    |
+| cobalt     | Cobalt    |
+| clover     | Clover    |
+| lemon      | Lemon     |
+| white      | White     |
+
+###### Item Assets Structure
+
+| Field              | Type   | Description                                                        |
+| ------------------ | ------ | ------------------------------------------------------------------ |
+| static_image_url   | string | The URL for the static image of the collectible                    |
+| animated_image_url | string | The URL for the animated image of the collectible (in APNG format) |
+| video_url?         | string | The URL for the video of the collectible                           |
+
+### Shop Block Object
```

Comment:
same nit as the above tagged union. move shop block type to the top and link each structure in the name field.
also make sure `type` is the first field in each structure for clarity.